ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/BlockProposalUtil.java

```diff
@@ -92,11 +95,13 @@ public BeaconBlock createEmptyBlock(
       final BeaconState previousState,
       final Bytes32 parentBlockSigningRoot)
       throws StateTransitionException {
+    final UnsignedLong newEpoch = compute_epoch_at_slot(newSlot);
     return createNewBlock(
         signer,
         newSlot,
         previousState,
         parentBlockSigningRoot,
+        StartupUtil.get_eth1_data_stub(previousState, newEpoch),
```

Comment:
What if we just reuse the previous eth1data instead of making something up?:
```suggestion
        previousState.getEth1_data(),
```

Reply:
The get_eth1_data_stub method was built for interop, to easily mock an eth1 data stub, we could totally get eth1 data from the previous state. However, I don't think it would add any value to change it, and it might introduce problems if we were to try interop testing again.